### PR TITLE
requirePaddingNewLinesBeforeLineComments: fix for comment on 2nd line…

### LINKED ISSUE #1527: Error processing rule
```
Error running rule requirePaddingNewLinesBeforeLineComments: This is an issue with JSCS and not your codebase.
Please file an issue (with the stack trace below) at: https://github.com/jscs-dev/node-jscs/issues/new
TypeError: Cannot read property 'type' of undefined
    at /Users/daniel.heath/Projects/ask-izzy/ask-izzy/node_modules/jscs/lib/rules/require-padding-newlines-before-line-comments.js:100:26
    at /Users/daniel.heath/Projects/ask-izzy/ask-izzy/node_modules/jscs/lib/js-file.js:483:17
    at Array.forEach (native)
    at Object.JsFile.iterateTokensByType (/Users/daniel.heath/Projects/ask-izzy/ask-izzy/node_modules/jscs/lib/js-file.js:481:26)
    at Object.module.exports.check (/Users/daniel.heath/Projects/ask-izzy/ask-izzy/node_modules/jscs/lib/rules/require-padding-newlines-before-line-comments.js:86:14)
    at null.<anonymous> (/Users/daniel.heath/Projects/ask-izzy/ask-izzy/node_modules/jscs/lib/string-checker.js:165:22)
    at Array.forEach (native)
    at StringChecker._checkJsFile (/Users/daniel.heath/Projects/ask-izzy/ask-izzy/node_modules/jscs/lib/string-checker.js:161:31)
    at StringChecker.checkString (/Users/daniel.heath/Projects/ask-izzy/ask-izzy/node_modules/jscs/lib/string-checker.js:106:14)
    at null.<anonymous> (/Users/daniel.heath/Projects/ask-izzy/ask-izzy/node_modules/jscs/lib/checker.js:44:21) at ./webpack/prod.load.js :
     1 |
--------^
     2 |// Register babel to have ES6 support
     3 |require("babel/register");
```

The file it's failing on is:
```

// Register babel to have ES6 support
require("babel/register");
module.exports = require("./prod.config");
```

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/24290844-error-processing-rule?utm_campaign=plugin&utm_content=tracker%2F281640&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F281640&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by hzoo**
… with blank line on 1st line

Fixes #1527, #1444

I'm just returning if `prevToken` is undefined.

### CODE REVIEW

File: lib/rules/require-padding-newlines-before-line-comments.js [hunk 1]
```diff
@@ -97,7 +97,7 @@ module.exports.prototype = {
 
             var prevToken = file.getPrevToken(comment, {includeComments: true});
 
-            if (prevToken.type === 'Line') {
+            if (!prevToken || prevToken.type === 'Line') {
```
Comment (hzoo) [inline lib/rules/require-padding-newlines-before-line-comments.js]: Maybe something else should be done - it just returns if prevToken is undefined

File: test/specs/rules/require-padding-newlines-before-line-comments.js [hunk 2]
```diff
@@ -66,6 +66,10 @@ describe('rules/require-padding-newlines-before-line-comments', function() {
             assert(checker.checkString('// comment\nvar a = 2;').isEmpty());
         });
 
+        it('should not report missing padding if newline is 1st line and the comment is 2nd line #1527', function() {
+            assert(checker.checkString('\n// comment\nvar a = 2;').isEmpty());
```
Comment (mrjoelkemp) [inline test/specs/rules/require-padding-newlines-before-line-comments.js]: Would this not work generically for any number of padding newlines before the first comment?

Comment (hzoo) [inline test/specs/rules/require-padding-newlines-before-line-comments.js]: No it would - yeah that's a bad description

**Comment by mrjoelkemp** [comment]
LGTM

**Comment by mrjoelkemp** [comment]
Nice work @hzoo

**Pull request closed without merging**